.gitignore

```diff
@@ -91,7 +91,6 @@ asv_bench/pandas/
 # Documentation generated files #
 #################################
 doc/source/generated
-doc/source/_static
```

Comment:
Can you leave this one? Many figures are created during doc building that are put inside of this, that otherwise show up in your git status view (files that have to be added to git just have to be force added then)